demo/scripts/getting_started_guide/04-publish-service.sh

```diff
@@ -52,7 +52,7 @@ spec:
   http:
   - name: bookinfo
     port: 80
-    hostname: bookinfo.tse.tetratelabs.io
+    hostname: bookinfo.$cluster_name.aws-ce.sandbox.tetrate.io
```

Comment:
$cluster_name.$external_dns_aws_dns_zone or similar to it

Reply:
what about annotations do we need to use any https://docs.tetrate.io/service-express/integrations/route53#annotations-supported-by-the-route53-controller ?